### DISCUSSION TITLE
Unclear benefit of modal manager

### DISCUSSION THREAD

**Discussion by Haberkamp** [mantinedev/mantine#2925] [Q&A] [answered]
Hello 👋 

can someone give me an explanation on when to use the modal manager instead of displaying a modal component? Because, from what I know is that the modal manager does not offer every features the modal component provides me like setting the size of an modal. 

Is there a use case when one should prefer using the modal manager of the modal component?

Thanks!

**Comment by rtivital** [comment, marked as answer]
I use modals manager mostly for confirmation modals, it is a quick way to open a modal with a single function call. Also you can pass all Modal component props to the function and they will be applied to the Modal component, for example, `openModal({ size: 700 })`

**Reply by Haberkamp** [reply]
Ahh okay, somehow I overlooked that. Thank you! 👍
